docs/documentation/siddhi-4.0.md

```diff
@@ -46,6 +46,208 @@ by a semicolon `( ; )` as shown in the below syntax.
+           	count = 10
+       	mean rate = 0.19 events/second
+   	1-minute rate = 0.11 events/second
+   	5-minute rate = 0.03 events/second
+  	15-minute rate = 0.01 events/second
+  org.wso2.siddhi.SiddhiApps.TestMetrics.Siddhi.Streams.TestStream.throughput
+           	count = 0
+       	mean rate = 0.00 events/second
+   	1-minute rate = 0.00 events/second
+   	5-minute rate = 0.00 events/second
+  	15-minute rate = 0.00 events/second
+  org.wso2.siddhi.SiddhiApps.TestMetrics.Siddhi.Trigger.FiveMinTriggerStream.throughput
+           	count = 10
+       	mean rate = 0.19 events/second
+   	1-minute rate = 0.11 events/second
+   	5-minute rate = 0.03 events/second
+  	15-minute rate = 0.01 events/second
+  
+  -- Timers ----------------------------------------------------------------------
+  org.wso2.siddhi.SiddhiApps.TestMetrics.Siddhi.Queries.logQuery.latency
+           	count = 10
+       	mean rate = 0.19 calls/second
+   	1-minute rate = 0.11 calls/second
+   	5-minute rate = 0.03 calls/second
+  	15-minute rate = 0.01 calls/second
+             	min = 0.38 milliseconds
+             	max = 1.22 milliseconds
+            	mean = 0.60 milliseconds
+          	stddev = 0.22 milliseconds
+          	median = 0.49 milliseconds
+            	75% <= 0.61 milliseconds
+            	95% <= 1.22 milliseconds
+            	98% <= 1.22 milliseconds
+            	99% <= 1.22 milliseconds
+          	99.9% <= 1.22 milliseconds
+  org.wso2.siddhi.SiddhiApps.TestMetrics.Siddhi.Queries.query1.latency
+           	count = 10
+       	mean rate = 0.19 calls/second
+   	1-minute rate = 0.11 calls/second
+   	5-minute rate = 0.03 calls/second
+  	15-minute rate = 0.01 calls/second
+             	min = 0.47 milliseconds
+             	max = 2.86 milliseconds
+            	mean = 0.84 milliseconds
+          	stddev = 0.57 milliseconds
+          	median = 0.66 milliseconds
+            	75% <= 0.74 milliseconds
+            	95% <= 2.86 milliseconds
+            	98% <= 2.86 milliseconds
+            	99% <= 2.86 milliseconds
+          	99.9% <= 2.86 milliseconds
+  org.wso2.siddhi.SiddhiApps.TestMetrics.Siddhi.StoreQueries.query.latency
+           	count = 0
+       	mean rate = 0.00 calls/second
+   	1-minute rate = 0.00 calls/second
+   	5-minute rate = 0.00 calls/second
+  	15-minute rate = 0.00 calls/second
+             	min = 0.00 milliseconds
+             	max = 0.00 milliseconds
+            	mean = 0.00 milliseconds
+          	stddev = 0.00 milliseconds
+          	median = 0.00 milliseconds
+            	75% <= 0.00 milliseconds
+            	95% <= 0.00 milliseconds
+            	98% <= 0.00 milliseconds
+            	99% <= 0.00 milliseconds
+          	99.9% <= 0.00 milliseconds
+
+</details>
+
+The `@app` annotation can be added to a Siddhi application as follows  to improve its performance.
+
+* `@app:async`: This annotation adds a disruptor to allow the Siddhi application to process events in parallel via multiple threads. 
```

Comment:
This is not possible in Siddhi 4.0.0. Stream definitions can include @Async annotation but not the whole Siddhi app